views/layouts/main.handlebars

```diff
@@ -6,6 +6,16 @@
   <meta name="defaultLanguage" content="en-US">
   <meta name="availableLanguages" content="{{availableLanguages}}">
 
+  <meta property="og:title" content="Firefox Send"/>
+  <meta name="twitter:title" content="Firefox Send"/>
+  <meta name="description" content="Encrypt and send files with a link that automatically expires to ensure your important documents don’t stay online forever."/>
+  <meta property="og:description" content="Encrypt and send files with a link that automatically expires to ensure your important documents don’t stay online forever."/>
+  <meta name="twitter:description" content="Encrypt and send files with a link that automatically expires to ensure your important documents don’t stay online forever."/>
+  <meta name="twitter:card" content="summary"/>
+  <meta property="og:image" content="/resources/send-fb.jpg"/>
+  <meta name="twitter:image" content="/resources/send-twitter.jpg"/>
+  <meta property="og:url" content="https://send.firefox.com"/>
```

Comment:
@dannycoates Does the Express server know whether it's serving as "https://send.firefox.com", "send.stage.mozaws.net", or "send.dev.mozaws.net"?

I think our only supported `NODE_ENV` values are "production", "development", and "test" (per [/server/config.js:35-39](https://github.com/mozilla/send/blob/80db158ee3de89cfe2de302814d7825d0a27fbd9/server/config.js#L35-L39)), so not sure I could use that to detect a difference between dev/stage.